bluesky/run_engine.py

```diff
@@ -2011,13 +2067,17 @@ def subscribe(self, func, name='all'):
         func: callable
             expecting signature like ``f(name, document)``
             where name is a string and document is a dict
-        name: {'start', 'descriptor', 'event', 'stop', 'all'}, optional
-            Defaults to 'all'
+        name: string
+            one of {'start', 'descriptor', 'event', 'stop', 'all'}
```

Comment:
https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt see section 4 the convention is to list
the valid values (and the default should go first).